src/main/java/io/openliberty/tools/intellij/util/LibertyGradleUtil.java

```diff
@@ -230,7 +230,8 @@ private static String getLocalGradleWrapperPath(String wrapperDir) throws Libert
         } catch (IOException e) {
             throw new LibertyException("Could not get canonical path for gradle wrapper file");
         }
-        return path;
+        String additionalCMD = SystemInfo.isWindows ? "cmd /K " : ""; // In Windows, "cmd /K" is used to execute a command
+        return additionalCMD + LibertyProjectUtil.includeEscapeToString(path);
```

Comment:
I see where you got this code but I find these two lines too complex. Also we have to do an unnecessary concatenation operation on line 234 for mac and linux. How about a more conventional:
String cmd = LibertyProjectUtil.includeEscapeToString(path);
if (SystemInfo.isWindows) {
  cmd = "cmd /K " + cmd;
}
return cmd;

Reply:
Thanks @turkeylurkey for the suggestion , I modified the code.